README.md

```diff
@@ -1,6 +1,15 @@
 # Log4jHotPatch
 
-This is a tool which injects a Java agent into a running JVM process. The agent will attempt to patch the `lookup()` method of all loaded `org.apache.logging.log4j.core.lookup.JndiLookup` instances to unconditionally return the string "Patched JndiLookup::lookup()". It is designed to address the [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228/) remote code execution vulnerability in Log4j without restarting the Java process. This tool will also address [CVE-2021-45046](https://nvd.nist.gov/vuln/detail/CVE-2021-45046/).
+This is a tool which injects a Java agent into a running JVM process. The agent will attempt to patch the `lookup()` 
+method of all loaded `org.apache.logging.log4j.core.lookup.JndiLookup` instances to unconditionally return the string 
+"Patched JndiLookup::lookup()". It is designed to address the 
+[CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228/) remote code execution vulnerability in Log4j without 
+restarting the Java process. This tool will also address 
+[CVE-2021-45046](https://nvd.nist.gov/vuln/detail/CVE-2021-45046/).
+
+To Patch for [CVE-2021-45105](https://nvd.nist.gov/vuln/detail/CVE-2021-45105), you can run the tool with the following
```

Comment:
Please expand on when/why some one would need to patch for this and what the side effects of it are.